test/flutter_inapp_purchase_test.dart

```diff
@@ -234,12 +248,6 @@ void main() {
                 expectedProduct.introductoryPriceSubscriptionPeriodIOS);
             expect(product.subscriptionPeriodAndroid,
                 expectedProduct.subscriptionPeriodAndroid);
-            expect(product.introductoryPriceCyclesAndroid,
-                expectedProduct.introductoryPriceCyclesAndroid);
-            expect(product.introductoryPricePeriodAndroid,
-                expectedProduct.introductoryPricePeriodAndroid);
-            expect(product.freeTrialPeriodAndroid,
-                expectedProduct.freeTrialPeriodAndroid);
```

Comment:
Are they being deprecated 🧐?

Reply:
They have been deprecated yes, these were essentially a way to make it look like an ios purchase but it's changed too much under the hood to be functional with the new Android billing